.github/actions/cluster-init/action.yaml

```diff
@@ -0,0 +1,102 @@
+name: Cluster Init
+description: Set up Turing Cluster Init
+
+inputs:
+  cluster_name: 
+    required: true
+    description: 'Name of Cluster'
+    default: ''
+  istio_version:
+    required: true
+    description: 'Istio Version'
+    default: ''
+  knative_version:
+    required: true
+    description: 'Knative Version'
+    default: ''
+  knative_istio_version:
+    required: true
+    description: 'Knative Istio Version'
+    default: ''
+  local_registry:
+    required: true
+    description: 'Endpoint of local registry'
+    default: ''
+  build_image:
+    required: false
+    description: '(Optional) Flag to build image from code'
+    default: false
```

Comment:
Since we are already building `cluster-init` image in Turing CI workflow, I would rather unify the flow and make it similar to Turing CI. To achieve that, we need:
1. Remove `build_image` input option from this action
2. Add explicit step `build-cluster-init` into `cluster-init` workflow, that builds the image and stores it as `.tar` artifact
3. Make `cluster-init` action to import that docker image into a local registry (it's already done, by using `cluster_init_version` variable)
4. Update `publish` step of the `cluster-init` workflow to load the Docker image from `.tar` artefact, instead of building it again with `make build-image`.